src/app/Library/CrudPanel/Traits/Search.php

```diff
@@ -109,8 +110,15 @@ public function applyDatatableOrder()
                 $column_direction = (strtolower((string) $order['dir']) == 'asc' ? 'ASC' : 'DESC');
                 $column = $this->findColumnById($column_number);
                 if ($column['tableColumn'] && ! isset($column['orderLogic'])) {
-                    // apply the current orderBy rules
-                    $this->orderByWithPrefix($column['name'], $column_direction);
+                    if (method_exists($this->model, 'translationEnabled') &&
+                        $this->model->translationEnabled() &&
+                        $this->model->isTranslatableAttribute($column['name']) &&
+                        is_a($this->model->getConnection()->getDoctrineColumn($this->model->getTableWithPrefix(), $column['name'])->getType(), JsonType::class)
```

Comment:
This PR should not be merged before the Schema (https://github.com/Laravel-Backpack/CRUD/pull/4649) since we need to remove this `getDoctrineColumn` call and get the column from the schema.